src/script/run_mypy.sh

```diff
@@ -10,31 +10,22 @@ python3 -m venv .mypy_venv
 pip install $(find * -name requirements.txt | grep -v node_modules | awk '{print "-r  " $0}')
 pip install mypy
 
-MYPY_INI=$(cat <<-EOF
-[mypy]
-strict_optional = True
-no_implicit_optional = True
-ignore_missing_imports = True
-warn_incomplete_stub = True
-check_untyped_defs = True
-show_error_context = True
-EOF
-)
+MYPY_INI="$PWD"/mypy.ini
```

Comment:
Why not putting `mypy.ini` also in the same dir as the script? Just checked `src/` and given is a top-level dir I'd not put anything else there. I've seen other ini files are placed within per-tool subdirs (e.g.: `src/ceph-volume/ceph-volume.ini`).

Reply:
The place was initial consensus with @sebastian-philipp ,
but as long as we reach new consensus, the place of mypy.ini can be updated.

Reply:
The idea (at least my hope) is that all python files in `src/` are tested with mypy eventually. 

* `ceph-volume/`,
* all tools in `tools/`,
* all tests in `tests/`,
* `telemetry/server/ceph_telemetry`,
* `script/`
* `pybind`
* `blkin/`
* `dmclock/benchmark/data_parser.py`

---
My thinking was:

* `src/` would be the common root for all use cases. That was my initial idea.
* `scripts/run_mypy.sh` is very specialized. Tox doesn't need to execute that script. So the .ini should be independent of run-mypy.sh